flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -322,11 +329,22 @@ public Builder cleanupInRocksdbCompactFilter() {
 		 */
 		@Nonnull
 		public Builder cleanupInRocksdbCompactFilter(long queryTimeAfterNumEntries) {
-			cleanupStrategies.activate(CleanupStrategies.Strategies.ROCKSDB_COMPACTION_FILTER,
+			strategies.put(CleanupStrategies.Strategies.ROCKSDB_COMPACTION_FILTER,
```

Comment:
Could you move to the new line `CleanupStrategies.Strategies.ROCKSDB_COMPACTION_FILTER,`?
According to the code style, we move all arguments to the new line if the line is broken.